shop/rest/renderers.py

```diff
@@ -3,6 +3,46 @@
 
 from rest_framework import renderers
 from rest_framework.compat import template_render
+from rest_framework.exceptions import APIException
+
+from shop.models.cart import CartModel
+from shop.serializers.cart import CartSerializer
+
+
+class ShopTemplateHTMLRenderer(renderers.TemplateHTMLRenderer):
+    """
+    Modified TemplateHTMLRenderer, which shall be used to render templates used by django-SHOP.
+    Instead of polluting the template context with the serialized data, that information is
+    stored inside a separate `data` attribute, which allows to add a Cart and Paginator object.
+
```

Comment:
This means that the behaviour of this renderer deviates considerably from standard DRF behaviour, so templates written for the DRF `TemplateHTMLRenderer` will be incompatible with `ShopTemplateHTMLRenderer`.

Can we keep the standard DRF behaviour of putting the serialized data into the template context and just add the cart and the paginator in addition? Or can we maybe even just use a context processor instead of a custom renderer?